common/src/main/resources/messages/messages.properties

```diff
@@ -670,6 +670,7 @@ stack.lb.update.create.entity=Creating load balancer entities.
 stack.lb.update.create.cloud.resource=Creating load balancer cloud resources.
 stack.lb.update.collect.metadata=Collecting load balancer metadata.
 stack.lb.update.register.public.dns=Registering load balancer public DNS.
+stack.lb.update.register.public.dns.failed=Load balancer public DNS registration failed. This has to be done manually. It's possible that instances are not reachable.
```

Comment:
Reword to say:

`Load balancer public DNS registration failed. It's possible that instances are not reachable, unless registration is manually done. Reach out to support if necessary.`

Reply:
Hey @biharitomi - 
There was an escalation about it: https://jira.cloudera.com/browse/ENGESC-20477?focusedCommentId=5961300&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-5961300
When a problem occurs during the resizing that the DNS entry wasn't updated.

Reply:
But what ENGESC has linked here is related to a FreeIPA DNS entry which is not managed by the modified logic and I think if the issue is really the FreeIPA DNS entry then it should be fixed programmatically.
If we really would like to go with a handy error message, then we should create and link our official documentation what steps need to be executed and where manually to solve this case.

Reply:
I mean [this logic](https://github.com/hortonworks/cloudbreak/pull/14994/files#diff-c71691ed7dc485b1dcfbad8cf8d089be88cd03344396440019095805c01d9ec2R140) is related to publicly available, Cloudera managed DNS entry management and not for the one which is mentioned by the ENGESC, because that is a DNS Entry re-add of FreeIpa which is managed in a different part of our code base.

Reply:
@biharitomi I think the intent was to ask reach out Cloudera support so that support can manually update the PEM.
@sxxgrc Can you comment on this?

Reply:
Yes the intent is for support to update the PEM for the customer. A simple fix which will address the core issue, as reflected in the linked escalation.

Reply:
But the message still contains only "This has to be done manually." please update it to be more details what the customer needs to do.

Reply:
Hey @biharitomi @sxxgrc - Rephased the message.